### DISCUSSION TITLE
Does ray show the embeded images of an email?

### DISCUSSION THREAD

**Discussion by ricklambrechts** [spatie/laravel-ray#166] [Q&A]
Hi,

I embedded 2 images in a mailable and for some quick testing i was dumping the mailable to ray.
The email shows but the embedded images appear as broken.

Does ray support embedded images of a mailable?

The mailable returns a view on build and inside the view i use:
`<img src="{{ $message->embed('path here') }}">`

**Comment by freekmurze** [comment]
It should work if you use an url to a to your computer accessible image.

**Comment by ricklambrechts** [comment]
I did some debugging and sent the raw response of `MailablePayload::forMailable(...)` to ray.

In the HTML of the payload, the img is `<img src="cid:973b7bed18eff409bd312df21679edaa@swift.generated">`.
But in the payload i don't see any attachments.

BTW. This package just uses the render function on the mailable so its definitely not an issue of this package.
I will look further into it.
